sdk/Resource/ResourceInfo.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Sdk\Resource;
+
+use OpenTelemetry\Sdk\Trace\Attributes;
+
+/**
+ * A Resource is an immutable representation of the entity producing telemetry. For example, a process producing telemetry
+ * that is running in a container on Kubernetes has a Pod name, it is in a namespace and possibly is part of a Deployment
+ * which also has a name. All three of these attributes can be included in the Resource.
+ */
```

Comment:
It might be prudent to make a note about how we are using `ResourceInfo` rather than `resource` here because of the reserved PHP word.